nengo/utils/builder.py

```diff
@@ -177,8 +177,8 @@ def _create_replacement_connection(c_in, c_out):
     return c
 
 
-def remove_passthrough_nodes(objs, connections,  # noqa: C901
-        create_connection_fn=_create_replacement_connection):
```

Comment:
Oh yeah, I did that because without it it would break onto 3 lines and there was no way to make it look good.